app/views/analytics/districts/show.csv.erb

```diff
@@ -88,7 +88,7 @@ end) %>
     *dates.map { |period| analytics_totals(@dashboard_analytics, :registered_patients_by_period, period) }
   ]
 %>
-<% policy_scope([:cohort_report, Facility.order(:name)]).each do |facility| %>
+<% current_admin.accessible_facilities(:view_reports).order(:name).each do |facility| %>
```

Comment:
I guess this bit hasn't been tested in production yet. Let's test it out well.

Reply:
👍